pkg/pub_package_reader/lib/pub_package_reader.dart

```diff
@@ -44,7 +44,7 @@ class PackageSummary {
   final List<String>? libraries;
 
   PackageSummary({
-    this.issues,
+    List<ArchiveIssue>? issues,
```

Comment:
You could also do: `issues = []` (I think)...

But this is fine too... The benefit of doing the other thing is that you can't accidentally pass null as `issues`.